onnxruntime/test/shared_lib/test_runtime_path.cc

```diff
@@ -46,5 +47,28 @@ TEST(GetRuntimePathFromSharedLibraryTest, Basic) {
 
   ASSERT_TRUE(std::filesystem::is_regular_file(canonical_shared_library_path));
 }
+#else
+TEST(GetRuntimePathFromSharedLibraryTest, Basic) {
```

Comment:
if GetRuntimePath() returns an empty path, it is not a valid one. perhaps the documentation needs to be updated, but I believe at least one place is relying on the returned path to be absolute.

https://github.com/microsoft/onnxruntime/blob/760eea48e1870c3c8164c50e806c6ec7ae49954f/onnxruntime/core/session/utils.cc#L419-L427

if it is unsupported on AIX, I think it's not worth testing that it returns an empty path and we can just skip this test.